package.json

```diff
@@ -3,7 +3,7 @@
   "version": "55.1.0",
   "main": "index.js",
   "author": "Salesforce",
-  "bugs": "https://github.com/forcedotcom/salesforcedx-templates/issues",
+  "bugs": "https://github.com/salesforcecli/plugin-templates/issues",
   "dependencies": {
     "@oclif/core": "^1.12.0",
     "@salesforce/command": "^5.2.1",
```

Comment:
Update to `tslib ^2` please (Github will not let me select that line for a comment 🤔)